app/src/main/res/values-de/strings.xml

```diff
@@ -1,3 +1,11 @@
 <resources>
     <string name="rc_sdk_version">SDK version: </string>
+    <string name="user_identifier">User Identifier</string>
+    <string name="username">Username</string>
+    <string name="company">Company</string>
+    <string name="email">Email</string>
+    <string name="firstname">Firstname</string>
+    <string name="lastname">Lastname</string>
+    <string name="home_phone">Home Phone</string>
+    <string name="mobile_phone">Mobile Phone</string>
```

Comment:
these are not translated, we can remove them and it should use the english ones by default, right?

Reply:
6f72596868b6b6b66a401a0ef859e5b02779ac7a